packages/fxa-settings/src/lib/gql.ts

```diff
@@ -8,27 +8,19 @@ import { ErrorHandler, onError } from '@apollo/client/link/error';
 import { BatchHttpLink } from '@apollo/client/link/batch-http';
 import { cache, sessionToken, typeDefs } from './cache';
 
-const pagesViewableWithoutAuthentication = [
```

Comment:
Changes to `gql.ts` were cherry picked in from https://github.com/mozilla/fxa/pull/15054 to allow testing on localhost until that PR is merged into main.

Reply:
thank you!!